### PR TITLE
docs(animations): Added consistency in code examples

### PR DESCRIPTION
**Pull request by Advaith3600**
Fixed missing quotes and comma in the documentation explanation code

### CODE REVIEW

**Comment by Advaith3600** [comment]
@googlebot I signed it!

File: aio/content/guide/route-animations.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Let's assume that we are routing from the *Home => About*.
 
 The animation code does the following after styling the views:
 
-* `query(':enter style({ left: '-100%'})` matches the view that is added and hides the newly added view by positioning it to the far left.
+* `query(':enter', style({ left: '-100%'})` matches the view that is added and hides the newly added view by positioning it to the far left.
```
Comment (gkalpak) [inline aio/content/guide/route-animations.md:114]: There is a missing `)` at the end btw:

```suggestion
* `query(':enter', style({ left: '-100%'}))` matches the view that is added and hides the newly added view by positioning it to the far left.
```

Also, the example has `style()` inside an array: `query(':enter', [style({ left: '-100%'})])` (which I am not sure is necessary, but we should align the actual example code and the code in this bullet point).

Finally, if you feel like fixing all the inconsistent spacing in object literals (such as `{ left: '-100%'}` --> `{ left: '-100%' }`), I won't stop you (in the guide and in the example code) 😉

**Review by gkalpak — CHANGES_REQUESTED** [review]
Good catch!

File: aio/content/guide/route-animations.md (hunk 1 above)
Comment (Advaith3600) [reply aio/content/guide/route-animations.md:114]: Challenge Accepted

**Comment by Advaith3600** [comment]
Added consistency in spacing as well 🥂

**Comment by Advaith3600** [comment]
Will this PR even be merged?

**Review by gkalpak — APPROVED** [review]
Sorry for the late response, @Advaith3600. I didn't realize this required my review 😇
Could you rebase this on master and squash the commits into one with a commit message that starts with `docs(animations): ...`?

**Comment by kapunahelewong** [comment]
@Advaith3600 I have some [notes on the repo workflow for rebasing, etc](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase). in case you'd like them. If you have any questions or would like me to do it from this end, just tag me. Thanks!

**Comment by Advaith3600** [comment]
Thanks for reminding me of doing that. I almost forgot about PR. Thanks!

**Comment by Advaith3600** [comment]
@gkalpak I did everything from my part 🥂

**Comment by Advaith3600** [comment]
@googlebot I consent

**Comment by Advaith3600** [comment]
@googlebot I consent.

**Comment by gkalpak** [comment]
@Advaith3600, this doesn't seem to be rebased on latest master (which causes CI to fail). Also, it seems you added some new changes to this PR that have already been fixed in #37897. Can you please rebase on latest master?

**Comment by Advaith3600** [comment]
@gkalpak Just rebased the PR with the master, and also made sure all the tests pass

**Review by gkalpak — APPROVED** [review]
Awesome! Thank you, @Advaith3600 :+1:

Reviewed-for: global-docs-approvers

**Pull request closed without merging**